src/Sylius/Bundle/ApiBundle/Resources/config/services/event_handlers.xml

```diff
@@ -17,7 +17,7 @@
 >
     <services>
         <service id="Sylius\Bundle\ApiBundle\EventHandler\OrderCompletedHandler">
-            <argument type="service" id="messenger.default_bus" />
+            <argument type="service" id="sylius_default.bus" />
```

Comment:
Looks like a good change - just a note that we probably should work on the naming of these buses. It doesn't follow convention (`sylius_default.bus` instead of `sylius.default_bus` or `sylius.bus.default`) and it's also a default bus for API Bundle (and it's defined there).